.travis.yml

```diff
@@ -4,8 +4,9 @@ rvm:
   - 2.2.2
   - 2.3.1
   - jruby-head
-branches:
-  only: [master]
+# TODO: turn back on once PR is finished
+#branches:
+#  only: [master]
```

Comment:
strange, that was needed ?

Reply:
Sure. With that flag there, Travis won't build other branches and I need to be able to explicitly trigger a build/rebuild for my feature branch over in my fork. It was either disable that flag or start pushing fake commits into the PR.